s3migrate/shared.py

```diff
@@ -235,9 +235,9 @@ def get_mogile_files_from_database(database_url, limit=None):
         with connection.cursor() as cursor:
             # Order by dkey, which is deterministic but random.
             if limit is not None:
-                sql = f"SELECT * FROM file ORDER BY dkey LIMIT {limit}"
+                sql = f"SELECT * FROM file LIMIT {limit}"
             else:
-                sql = "SELECT * FROM file ORDER BY dkey"
```

Comment:
Weird that this was slow. I figured something with the word `key` in the name would almost certainly be indexed

Reply:
It is indeed, but it still slowed down the start quite a bit. Not sure why.

Reply:
hmm, weeerd